src/LGraph.ts

```diff
@@ -648,7 +650,8 @@ export class LGraph {
                 node.id = LiteGraph.uuidv4()
         }
         else {
-            if (node.id == null || node.id == -1) {
+            // LEGACY: This was changed from null check to non-number check
```

Comment:
This change is causing failure on loading workflow with string node id. Revert it for now?